app/views/katello/api/v2/hosts_bulk_actions/erratum.json.rabl

```diff
@@ -1,9 +1,11 @@
 extends "katello/api/v2/errata/show"
 
-node :applicable_hosts do |erratum|
-  erratum.hosts_available.where("#{::Host.table_name}.id" => @hosts).
-  select(["#{::Host.table_name}.id", "#{::Host.table_name}.name"]).
-  collect { |host| {:name => host.name, :id => host.id} }
+if @compatibility_mode
```

Comment:
Thought about this for a bit. Since it effectively breaks the API I was going to request that you add the param to the apipie spec as well as rename it to make its function more clear.

Then I thought some more and concluded that this data seems out of place in the response of this API - after all, you are hiding this node by default. I guess it was trying to do too much by making a convenient API for the UI view rather than a performant and well-scoped API.

My vote would be to undo the compatibility mode change and add a deprecation warning to the two controller methods stating that the applicable hosts data will not be available from the API response in Katello 2.0

@jlsherrill does this sound like a sane suggestion?

Reply:
@jturel I'm okay with that suggestion, however we may want to take over this pr as @ShimShtein i think is working on other stuff at the moment.